jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/test/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminViewTest.java

```diff
@@ -0,0 +1,257 @@
+        when(currentListGrid.getGridPreferencesStore()).thenReturn(gridPreferencesStoreMock);
+        when(gridPreferencesStoreMock.getColumnPreferences()).thenReturn(columnPreferences);
+
+        view.initColumns( currentListGrid );
+
+        verify( currentListGrid ).addColumns( anyList() );
+    }
+
+    @Test
+    public void initColumnsWithTaskVarColumnsTest() {
+        doAnswer( new Answer() {
+            @Override
+            public Void answer( InvocationOnMock invocationOnMock ) throws Throwable {
+                final List<ColumnMeta> columns = (List<ColumnMeta>) invocationOnMock.getArguments()[ 0 ];
+                assertTrue(columns.size()==17);
+                return null;
+            }
+        } ).when( currentListGrid ).addColumns(anyList());
+
+        ArrayList<GridColumnPreference> columnPreferences = new ArrayList<GridColumnPreference>();
+        columnPreferences.add(new GridColumnPreference( "var1",0,"40" ));
+        columnPreferences.add(new GridColumnPreference( "var2",1,"40" ));
+        columnPreferences.add(new GridColumnPreference("var3", 1, "40"));
+        when(currentListGrid.getGridPreferencesStore()).thenReturn(gridPreferencesStoreMock);
+        when(gridPreferencesStoreMock.getColumnPreferences()).thenReturn(columnPreferences);
+
+        view.initColumns( currentListGrid );
+
+        verify( currentListGrid ).addColumns( anyList() );
+    }
+
+    @Test
+    public void initDefaultFiltersOwnTaskFilter() {
+        view.initDefaultFilters(new GridGlobalPreferences(), mockButton);
+
+        verify(filterPagedTableMock, times(1)).addTab(any(ExtendedPagedTable.class), anyString(), any(Command.class));
+        verify(filterPagedTableMock).addAddTableButton(mockButton);
+        verify(presenter).setAddingDefaultFilters(true);
+        verify(presenter).setAddingDefaultFilters(false);
+    }
+
+    @Test
+    public void addDomainSpecifColumnsTest() {
+        final Set<String> domainColumns = new HashSet<String>();
+        domainColumns.add("var1");
+        domainColumns.add("var2");
+        domainColumns.add("var3");
+
+        view.addDomainSpecifColumns(currentListGrid, domainColumns);
+
+        final ArgumentCaptor<List> argument = ArgumentCaptor.forClass(List.class);
+        verify(currentListGrid).addColumns(argument.capture());
+
+        final List<ColumnMeta> columns = argument.getValue();
+        assertTrue(columns.size() == 3);
+        int i = 0;
+        for (String domainColumn : domainColumns) {
+            assertEquals(columns.get(i).getCaption(), domainColumn);
+            i++;
+        }
+
+    }
+
+    @Test
+    public void setDefaultFilterTitleAndDescriptionTest() {
+        view.resetDefaultFilterTitleAndDescription();
+
+        verify(filterPagedTableMock, times(1)).getMultiGridPreferencesStore();
+        verify(filterPagedTableMock, times(1)).saveTabSettings(anyString(), any(HashMap.class));
+    }
```

Comment:
Evaluation of condition in _resetDefaultFilterTitleAndDescription()_ call depends on Mockito default value for collections. Instead, could we be specific through additional stubbing in the setup method which return value will be used for condition evaluation?
Does is make sense to test the case when the condition evaluates to _true_ too?